Common/Brokerages/DefaultBrokerageModel.cs

```diff
@@ -149,6 +149,10 @@ public decimal GetLeverage(Security security)
             switch (security.Type)
             {
                 case SecurityType.Equity:
+                    if (AccountType == AccountType.Cash)
```

Comment:
Would the leverage==1 apply to more than just equity?

Reply:
Probably `SecurityType.Crypto`, when we merge this one: https://github.com/QuantConnect/Lean/pull/1098 ?

Reply:
Yes, most crypto exchanges offer cash and margin accounts. In some cases cash is the only option due to regulatory constraints.

Reply:
I guess my question really is... if the user selects cash account, then shouldn't it apply that to all security types instead of just equity?

Reply:
@mchandschuh agree. It is more of a brokerage level property. 

@dannypurcell thank you for the PR; & welcome to LEAN! -- Please move this to the top of the method & we can merge it in.

Reply:
Thanks! 
Done
Forex trading requires margin right? Do we already block Forex accounts from setting account type to Cash?

Reply:
Its conceivable people would trade forex with a cash account as well. Is there something else I'm missing specific to fx?

Reply:
More likely there is something I am missing. 
I only looked at Forex trading once, a long time ago. The brokerage I was using at the time only offered Forex to margin accounts. That does not seem to be the case for Oanda or IB.